src/Renderer/RendererOpenGL.cpp

```diff
@@ -579,6 +579,8 @@ void RendererOpenGL::size(int w, int h)
 void RendererOpenGL::minimum_size(int w, int h)
 {
 	SDL_SetWindowMinimumSize(underlyingWindow, w, h);
+	_resize(w, h);
+	SDL_SetWindowPosition(underlyingWindow, SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED);
 }
```

Comment:
Ok, looking back at this now, the call to `_resize` is fine. (It's just horribly named, and actually recalculates viewport dimensions to match window size after a resize event).

The unconditional use of `SDL_SetWindowPosition` still doesn't look right to me. It does makes sense to move a window if it overflows the screen. If it does move, it should be moved just enough to not clip, not necessarily to recenter it. If the window doesn't overflow the screen, it shouldn't be moved at all.

I would like to see this updated so it doesn't recenter the window on each resize. Alternatively, the window movement update can be deferred to a different PR, leaving this one to focus on just recalculating the viewport dimensions.